openmetadata-service/src/main/resources/json/data/kpi/kpiDefinitionsData.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "name": "EntityWithOwnership",
+    "displayName": "Entity with Ownership",
+    "fullyQualifiedName": "EntityWithOwnership",
+    "description": "80 percent of Entities should have Owners",
+    "dateInsightChart": "refrence to the chart for this kpi",
+    "target": [
+      {
+        "name": "completedDescriptionFraction",
+        "dataType": "PERCE",
+        "required": true
+      }
+    ],
+    "targetValue": [
+      {
+        "name": "completedDescriptionFraction",
+        "value": 0.08
+      }
+    ],
+    "owner": "Owner of this Definitions"
+  },
+  {
+    "name": "EntityWithDescription",
+    "displayName": "Entity with Description",
+    "fullyQualifiedName": "EntityWithDescriptions",
+    "description": "Entities should have Descriptions",
+    "metrics": [
+      {
+        "name": "percentComplete",
+        "dataType": "NUMBER",
+        "required": true
+      },
+      {
+        "name": "confidenceScore",
+        "dataType": "NUMBER",
+        "required": true
+      }
+    ],
+    "owner": "Owner of the Kpi def",
+  },
+  {
+    "name": "EntityWithTier",
+    "displayName": "Entity with Tier",
+    "fullyQualifiedName": "EntityWithTier",
+    "description": "Entities should have Tiers",
+    "metrics": [
+      {
+        "name": "percentComplete",
+        "dataType": "NUMBER",
```

Comment:
Should that be `PERCENTAGE`?

Reply:
this needs to be removed , don't need this json actually